TriblerGUI/widgets/trustpage/js/main.js

```diff
@@ -24,11 +32,15 @@ var state = {
     previous_angle: 0,
     previous_focus_pk: null,
```

Comment:
You could use the shorthand syntax: `previous_focus_pk = focus_node = user_pk = null`